app/controllers/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to root_path
+    else
```

Comment:
Once a new user is created successfully, you should redirect your app to the `new_user_path`